libs/sdk-ui-filters/tsconfig.build.esm.json

```diff
@@ -3,6 +3,8 @@
     "compilerOptions": {
         "module": "ESNext",
         "outDir": "esm",
-        "incremental": true
+        "incremental": true,
+        "declaration": true,
```

Comment:
This is not needed here, it will get inherited from tsconfig.build.json

Reply:
updated